stacks/spica/src/passport/pages/identity-add/identity-add.component.html

```diff
@@ -21,6 +21,7 @@ <h4>
         <mat-error *ngIf="identifierModel?.errors?.required">This field is required.</mat-error>
       </mat-form-field>
       <button
+        class="change-password"
```

Comment:
In many cases, you don't need a class selector at all. Can accomplish the thing with a more strict CSS selector.

For instance:   mat-card-actions button:last:-of-type  instead of button.change-password